kolibri/content/test/test_sqlalchemy_bridge.py

```diff
@@ -205,56 +197,6 @@ def test_field_no_class(self, ColumnDefaultMock, apps_mock, get_class_mock):
         ColumnDefaultMock.assert_not_called()
 
 
-@patch('kolibri.content.utils.sqlalchemybridge.BASE_CLASSES_CACHE')
```

Comment:
The `get_base` function no longer exists, because we explicitly get things from the `BASE` global variable now.